Tests/SonarScanner.MSBuild.PreProcessor.Test/PreProcessorTests.cs

```diff
@@ -82,9 +82,7 @@ public async Task Execute_CannotCreateDirectories_ReturnsFalseAndLogsError()
             using var lockedFile = new FileStream(Path.Combine(configDirectory, "LockedFile.txt"), FileMode.CreateNew, FileAccess.ReadWrite, FileShare.None);
 
             (await preProcessor.Execute(CreateArgs())).Should().BeFalse();
-            factory.Logger.AssertErrorLogged(
-@$"Failed to create an empty directory '{configDirectory}'. Please check that there are no open or read-only files in the directory and that you have the necessary read/write permissions.
-  Detailed error message: The process cannot access the file 'LockedFile.txt' because it is being used by another process.");
+            factory.Logger.Errors.Should().ContainMatch($"Failed to create an empty directory '{configDirectory}'. Please check that there are no open or read-only files in the directory and that you have the necessary read/write permissions.*  Detailed error message: The process cannot access the file 'LockedFile.txt' because it is being used by another process.");
```

Comment:
What's the problem with the old assertion? I liked it better than the `*` approach as you have to read the fluent assertion documentation to understand that `*` is a wildcard.

Reply:
The star replaces the end of the line. It seems to be different on my machine than on the server and I would like to have it green for both :)